src/api/domain/user-admin/user-admin.service.ts

```diff
@@ -12,40 +12,50 @@ import { Team } from '../team/models/team.entity';
 export class UserAdminService {
     constructor(
         @InjectRepository(Team)
-        private userRepository: Repository<Team>) {
+        private teamRepository: Repository<Team>) {
     }
 
     async findAll(pageRequest = new PageRequest()): Promise<Page<TeamDto>> {
 
         if (pageRequest.page === undefined) {
             pageRequest = new PageRequest();
         }
-        const [result, total] = await this.userRepository.findAndCount({
+        const [result, total] = await this.teamRepository.findAndCount({
             take: pageRequest.size,
             skip: pageRequest.page * pageRequest.size
         });
 
         return new Page<TeamDto>({
-            content: result.map(user => user.toDto()),
+            content: result.map(team => team.toDto()),
             page: pageRequest.page,
             size: pageRequest.size,
             totalElements: total,
         });
     }
 
+    async getTeamCount(): Promise<number> {
+        return (await this.teamRepository.count());
+    }
+
     async add(team: Team): Promise<TeamDto> {
-        return (await this.userRepository.save(team)).toDto();
+        return (await this.teamRepository.save(team)).toDto();
     }
 
     async findOne(id: string): Promise<TeamDto> {
-        return (await this.userRepository.findOne(id)).toDto();
+        return (await this.teamRepository.findOne(id)).toDto();
     }
 
     async update(id: number, team: Team) {
-        return this.userRepository.update(id, team);
+        return this.teamRepository.update(id, team);
     }
 
     async remove(id: string): Promise<void> {
-        await this.userRepository.delete(id);
+        await this.teamRepository.delete(id);
+    }
+
+    async validateTeam(id: number): Promise<void> {
+        let team = await this.teamRepository.findOne(id);
```

Comment:
jak nie zmieniasz to stosuj const :) po to są
```suggestion
        const team = await this.teamRepository.findOne(id);
```

Reply:
team.userValid = true to nie zmiana?